src/components/common/Tree/CustomTreeItem.js

```diff
@@ -52,25 +58,30 @@ const CustomTreeItem = ({ itemId, expandedItems = [], selectedId }) => {
     }
     const filteredChildren = children?.filter(
       (child, idx) =>
-        !isHidden(childrenTags?.get(idx)) && child.type === ITEM_TYPES.FOLDER,
+        !isHidden(childrenTags?.get(idx)) &&
+        [ITEM_TYPES.FOLDER, ITEM_TYPES.SHORTCUT].includes(child.type),
     );
 
     if (!filteredChildren?.size) {
       return null;
     }
 
-    return filteredChildren.map(({ id: childId }) => (
+    return filteredChildren.map((childItem) => (
       <CustomTreeItem
-        key={childId}
-        itemId={childId}
+        key={childItem.id}
         expandedItems={expandedItems}
         selectedId={selectedId}
+        itemProp={childItem}
       />
     ));
   };
 
   const content = childrenIsLoading ? LoadingTreeItem : item.name;
 
+  if (item.type !== ITEM_TYPES.FOLDER) {
```

Comment:
What is this case for? Would it make sense to move this check earlier, right after the `useItem` hook call, for performance? Does it also always make sense to return `null`, e.g. are there exceptional cases where displaying `<Alert severity="error"> ...` would make more sense? Let me know 😊

Reply:
We are loading the useItem + useItemTags. Now the item could not be a folder, because we are also passing shortcut items, so we have to filter and only render folders. I have moved the if statement a little before, but it is still the same because we are not loading more hooks (useChildren and useChildrenTags are not enabled if they are not folders) right before we return a null.